### DISCUSSION TITLE
Keys: Island Song (Adventure Time)

### DISCUSSION THREAD

**Discussion by JmDemisana** [Monika-After-Story/MonikaModDev#7805] [general - piano notes]
#6512 - Download Here

Come along with me
`2 W E Y R`

And the butterflies and bees
`Y U 9 U Y U U Y`

We can wander through the forest
`Y U 9 U Y U R Y `

And do so as we please
`W E Y Y 2 Q`

Come along with me
`2 W E Y R`

To a cliff under a tree
`Y U 9 U Y U U Y`

Where we can gaze upon the water
`Y U 9 U Y U R Y`

As an everlasting dream
`E R Y 6 U Y`

All of my collections
`9 9 9 U U Y`

I'll share them all with you
`E R Y Y 2 2`

Maybe by next summer
`9 O 9 U U Y`

We won't have changed our tunes
`W E Y Y 9 9 U`

We'll still want to be
`2 W E Y R`

With the butterflies and bees
`Y U 9 U Y U U Y`

Making up new numbers
`9 U Y U R Y`

And living so merrily
`E R Y Y U U Y`

All of my collections
`9 9 9 U U Y`

I'll share them all with you
`E R Y Y 2 2`

I'll be here for you always
`W 9 O 9 U U Y`

And always be with you
`W E Y Y 9 9 U`

Come along with me
`2 W E Y R`

And the butterflies and bees
`Y U 9 U Y U U Y`

We can wander through the forest
`Y U 9 U Y U R Y`

And do so as we please
`U 9 U Y U Y`

Living so merrily
`R Y Y 6 U Y`

#3765

**Comment by 20RaVen23** [comment]
niceu
